dask/dataframe/groupby.py

```diff
@@ -1702,79 +1740,139 @@ def cumcount(self, axis=no_default):
         )
 
     @derived_from(pd.core.groupby.GroupBy)
-    def sum(self, split_every=None, split_out=1, shuffle=None, min_count=None):
+    @numeric_only_deprecate_default
+    def sum(
+        self,
+        split_every=None,
+        split_out=1,
+        shuffle=None,
+        min_count=None,
+        numeric_only=no_default,
+    ):
+        numeric_kwargs = (
+            {} if numeric_only is no_default else {"numeric_only": numeric_only}
+        )
         result = self._single_agg(
             func=M.sum,
             token="sum",
             split_every=split_every,
             split_out=split_out,
             shuffle=shuffle,
+            chunk_kwargs=numeric_kwargs,
+            aggregate_kwargs=numeric_kwargs,
         )
         if min_count:
             return result.where(self.count() >= min_count, other=np.NaN)
         else:
             return result
 
     @derived_from(pd.core.groupby.GroupBy)
-    def prod(self, split_every=None, split_out=1, shuffle=None, min_count=None):
+    @numeric_only_deprecate_default
+    def prod(
+        self,
+        split_every=None,
+        split_out=1,
+        shuffle=None,
+        min_count=None,
+        numeric_only=no_default,
+    ):
+        numeric_kwargs = (
+            {} if numeric_only is no_default else {"numeric_only": numeric_only}
+        )
         result = self._single_agg(
             func=M.prod,
             token="prod",
             split_every=split_every,
             split_out=split_out,
             shuffle=shuffle,
+            chunk_kwargs=numeric_kwargs,
+            aggregate_kwargs=numeric_kwargs,
         )
         if min_count:
             return result.where(self.count() >= min_count, other=np.NaN)
         else:
             return result
 
     @derived_from(pd.core.groupby.GroupBy)
-    def min(self, split_every=None, split_out=1, shuffle=None):
+    def min(self, split_every=None, split_out=1, shuffle=None, numeric_only=no_default):
+        if numeric_only is no_default:
+            numeric_only = False
```

Comment:
Why aren't we using using `numeric_only_deprecate_default` and 

```python
        numeric_kwargs = (
            {} if numeric_only is no_default else {"numeric_only": numeric_only}
        )
```

here like we are in, for example, `prod`? This looks intentional, my guess is I'm just missing something

Reply:
Yeah, in Pandas 1.5, for `min`, `max`, `first`, and `last`, there's no deprecation warning, and default is `False`. This is recreating the same behavior.

Reply:
Thanks for clarifying 👍